extension/src/popup/components/mnemonicPhrase/ConfirmMnemonicPhrase.tsx

```diff
@@ -128,7 +128,7 @@ export const ConfirmMnemonicPhrase = ({
   return (
     <>
       <Formik initialValues={initialWordState} onSubmit={handleSubmit}>
-        {({ setValues, values, isSubmitting, handleChange }) => (
+        {({ dirty, setValues, values, isSubmitting, handleChange }) => (
```

Comment:
`dirty` checks if all fields have been altered from their initial state. Basically, has the user filled out the form yet
https://formik.org/docs/api/formik#dirty-boolean

Reply:
not a great/intuitive prop name. Also, kind of a bummer we need to use this render props pattern because we need to remember to pass this value every time we create a form